netci.groovy

```diff
@@ -559,6 +559,10 @@ combinedScenarios.each { scenario ->
                             switch (architecture) {
                                 case 'x64':
                                 case 'x86':
+
+                                    //This will shorten the output file path lengths
+                                    buildCommands += "set __IntermediateDir=int"
```

Comment:
Instead of d:\j\workspace\dotnet_coreclr\x64_release_windows_nt_pri1_bld\tests\..\bin\tests\obj\Windows_NT.x64.Release\Managed\..., you will now see d:\j\workspace\dotnet_coreclr\x64_release_windows_nt_pri1_bld\tests\..\bin\int\...

The "x64_release_windows_nt_pri_bld" in the pathname serves as a unique specifier.